plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/TestElasticsearchMetadata.java

```diff
@@ -13,15 +13,91 @@
  */
 package io.trino.plugin.elasticsearch;
 
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.io.BaseEncoding;
 import io.airlift.slice.Slices;
+import io.trino.plugin.elasticsearch.client.IndexMetadata;
+import io.trino.plugin.elasticsearch.client.TestingElasticsearchClient;
+import io.trino.spi.TrinoException;
+import io.trino.spi.connector.SchemaTableName;
+import io.trino.spi.type.TestingTypeManager;
+import org.intellij.lang.annotations.Language;
 import org.testng.annotations.Test;
 
+import java.util.Map;
 import java.util.Optional;
 
+import static io.trino.type.IpAddressType.IPADDRESS;
+import static java.lang.String.format;
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.assertj.core.api.Assertions.assertThatNoException;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static org.testng.Assert.assertEquals;
 
 public class TestElasticsearchMetadata
 {
+    @Test
+    public void testPassThroughQueryWithEnabledFeature()
+    {
+        @Language("JSON")
+        String query = "{\n" +
+                "    \"size\": 0,\n" +
+                "    \"aggs\" : {\n" +
+                "        \"max_orderkey\" : { \"max\" : { \"field\" : \"orderkey\" } },\n" +
+                "        \"sum_orderkey\" : { \"sum\" : { \"field\" : \"orderkey\" } }\n" +
+                "    }\n" +
+                "}";
+
+        String schemaName = "test_schema";
+        Map<String, IndexMetadata> indices = ImmutableMap.of(
+                schemaName,
+                new IndexMetadata(new IndexMetadata.ObjectType(ImmutableList.of())));
+        ElasticsearchMetadata elasticsearchMetadata = new ElasticsearchMetadata(
+                new TestingTypeManager(ImmutableList.of(IPADDRESS)),
```

Comment:
Use `PLANNER_CONTEXT.getTypeManager()` 

then you can revert `TestingTypeManager` changes